### PR TITLE
Update docs/howto.rst

### PR DESCRIPTION
**Pull request by Andrew-Chen-Wang**
## Description

Update docs/howto.rst after the PR that fixed docs building. I also wonder if we should add this "howto" portion to the README. Not sure. (if you need to check this, visit https://docs.hearye.us)

Checklist:

- [X] I've made sure that `tests/test_cookiecutter_generation.py` is updated accordingly (especially if adding or updating a template option)
- [X] I've updated the documentation or confirm that my change doesn't require any updates

## Rationale

The _source directory doesn't exist

### CODE REVIEW

File: {{cookiecutter.project_slug}}/docs/howto.rst [hunk 1]
```diff
@@ -4,20 +4,19 @@ How To - Project Documentation
 Get Started
 ----------------------------------------------------------------------
 
-Documentation can be written as rst files in the `{{cookiecutter.project_slug}}/docs/_source`.
+Documentation can be written as rst files in `{{cookiecutter.project_slug}}/docs`.
```
Comment (browniebroke) [inline {{cookiecutter.project_slug}}/docs/howto.rst:7]: I guess the intend of these `` ` `` is to have path formatted as `preformatted` not _italic_, which is done with double backtick ``` `` ``` in RST, single backtick work in Markdown... 

WDYT? Would you mind fixing them in this file?

**Review by browniebroke — APPROVED** [review]
LGTM

File: {{cookiecutter.project_slug}}/docs/howto.rst (hunk 1 above)
Comment (Andrew-Chen-Wang) [reply {{cookiecutter.project_slug}}/docs/howto.rst:7]: ah sorry I missed this comment! Yes, I agree. I can fix those in a separate pr.

Comment (browniebroke) [reply {{cookiecutter.project_slug}}/docs/howto.rst:7]: Yes, wan't directly related, so I decided to go ahead with it.

**Pull request merged**